bin/pa11y-ci.js

```diff
@@ -201,32 +201,26 @@ function defaultConfig(config) {
 
 // Load a sitemap from a remote URL, parse out the
 // URLs, and add them to an existing config object
-function loadSitemapIntoConfig(program, config) {
-	const sitemapUrl = program.sitemap;
+function loadSitemapIntoConfig(programWithSitemap, initialConfig) {
+	const sitemapUrl = programWithSitemap.sitemap;
 	const sitemapFind = (
-		program.sitemapFind ?
-		new RegExp(program.sitemapFind, 'gi') :
+		programWithSitemap.sitemapFind ?
+		new RegExp(programWithSitemap.sitemapFind, 'gi') :
 		null
 	);
-	const sitemapReplace = program.sitemapReplace || '';
+	const sitemapReplace = programWithSitemap.sitemapReplace || '';
 	const sitemapExclude = (
-		program.sitemapExclude ?
-		new RegExp(program.sitemapExclude, 'gi') :
+		programWithSitemap.sitemapExclude ?
+		new RegExp(programWithSitemap.sitemapExclude, 'gi') :
 		null
 	);
 
-	function getUrlsFromSitemap(sitemapUrl, config) {
+	function getUrlsFromSitemap(aSitemapUrl, config) {
```

Comment:
If `const sitemapUrl = programWithSitemap.sitemap;` is removed on line 205, then we can leave this named `sitemapUrl`, as variable names starting with articles looks cluttered.